webapp/src/pages/createApplicationForm/components/Question.js

```diff
@@ -359,9 +567,61 @@ const Question = forwardRef(({
                   ))}
                 </tbody>
               </table>
+              <span className='error-label'>{t(errorMessage)}</span>
           </div>
           )}
-          
+          {withReferals.includes(inputs.type && inputs.type.value) && (
+            <div className='referals-wrapper'>
+              <input
+                type='number'
+                min={0}
+                placeholder={t('Enter Minum Number of Referrals')}
+                className='referals-input'
+                value={inputs.options.min_num_referral}
+                onChange={handlereferralsChange('min')}
+               />
+              <input
+                type='number'
+                min={0}
+                placeholder={t('Enter Maximum Number of Referrals')}
+                className='referals-input'
+                onChange={handlereferralsChange('max')}
+                value={inputs.options.max_num_referral}
+               />
+              <div className='error-label'>{t(errorMessage)}</div>
+            </div>
+          )}
+          {withExtention.includes(inputs.type && inputs.type.value) && (
+            <div className='extensions-wrapper'>
+              <input
+                type='text'
+                placeholder={
+                  t('Please enter a list of file extensions, each starting with a period and separated with comas. E.g .csv,.xls')}
```

Comment:
Same comment as above regarding the placeholder, here the placeholder contains some valuable information. Suppose the user wants to edit an existing form that has ".xls" already entered, they won't know that they can add another separated by commas because this placeholder won't be visible...